testing/tests/DevExpress.ui.events/eventsInteraction.tests.js

```diff
@@ -602,6 +613,33 @@ QUnit.test('gesture should be started with wrong direction after timeout', funct
     assert.equal(swipeFired, 1, 'swipestart was fired');
 });
 
+QUnit.test('gesture should be started with wrong direction after timeout, immediateTimeout: 100', function(assert) {
+    let swipeFired = 0;
+    const $element = $('#element').on(swipeEvents.start, { immediate: true, immediateTimeout: 100, direction: 'horizontal' }, function() {
+        swipeFired++;
+    });
+    const pointer = pointerMock($element);
+
+    pointer.start().down().move(0, 1);
+    assert.strictEqual(swipeFired, 0, 'swipestart was fired');
+    this.clock.tick(90);
```

Comment:
let's combine rows like "action + assert + empty row"? it would be easier to read
```javascript
    pointer.start().down().move(0, 1);
    assert.strictEqual(swipeFired, 0, 'swipestart was fired');

    this.clock.tick(90);
    assert.strictEqual(swipeFired, 0, 'swipestart was not fired');

    this.clock.tick(10);
    pointer.move(0, 1);
    assert.strictEqual(swipeFired, 1, 'swipestart was fired');
});
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/23852/commits/d481540f0ec17b1cd5e2ef0857644c50e115c40c